website/notifications/emails.py

```diff
@@ -124,6 +126,11 @@ def check_node(node, event):
     return node_subscriptions
 
 
+def check_target_user(user, event):
```

Comment:
Oh, I see. It's not really filtering through subscriptions though so how about `get_user_subscription` instead?